HenHen.Framework/Graphics2d/Drawable.cs

```diff
@@ -18,6 +18,7 @@ public abstract class Drawable
         private Vector2 anchor;
         private Vector2 origin;
         private bool masking;
+        public bool Visible { get; set; } = true;
```

Comment:
Add documentation above this property
"Whether this drawable is rendered to the screen. True by default."
Powinna się zaczynać od trzech forward slashy /// i cały tekst powinien znajdować się w tagu <summary>